tethys_portal/routing.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import re_path
+from tethys_portal.views.app_lifecycle import AppLifeCycleConsumer
+
+
+websocket_urlpatterns = [
+    re_path(r"ws/app-lifecycle/(?P<app_name>\w+)/", AppLifeCycleConsumer.as_asgi()),
+]
```

Comment:
Move this to tethys_portal/urls.py unless there is a reason you need it in a separate file.

Reply:
Done.